From: Michael Tokarev <mjt@tls.msk.ru>
Date: Sun, 01 May 2022 01:26:08 +0300
Subject: no git version

The upstream build system derives package version from the git repository
information if .git/ exists, giving different results when building the
same source with or without .git/. Stop scripts/buildversion.py from looking
at .git/ entirely and always looking at file named ".version" as it does
when no .git/ exists.

diff --git a/scripts/buildversion.py b/scripts/buildversion.py
index 8875497c..a26dda2a 100755
--- a/scripts/buildversion.py
+++ b/scripts/buildversion.py
@@ -115,3 +115,3 @@ def main():
 
-    ver = git_version()
+    ver = ''
     cleanbuild = cleanbuild and 'dirty' not in ver
